azerothcore/azerothcore-wotlk
# fix(DB/quest_poi): Update WorldMapAreaId to match respective quests

## Changes Proposed:
-  Update WorldMapAreaId to match respective quests in issues linked below

## Issues Addressed:
- Closes #8531
- Closes #8713
- Closes #8770
- Closes #9085
- Closes chromiecraft/chromiecraft#2100
- Closes chromiecraft/chromiecraft#2175
- Closes chromiecraft/chromiecraft#2200

## How to Test the Changes:
1. Add the quests 894, 900, 901, 902, 890, 892, 896, 888, 887, 3924, 6164, 1125, 1126 and 6570 to your quest log
2. Right click on quest tracker and show map
3. Should now bring you to where the quest is located and not the next zone over
